### PR TITLE
Fix packaging

### PR DESCRIPTION
**Pull request by benjaoming**
#413

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -3,6 +3,16 @@
 Changelog
 =========
 
+
+`0.13.4`_ - 2018-05-12
```
Comment (Stranger6667) [inline docs/changes.rst]: It should be `2018-05-19`

Comment (benjaoming) [reply docs/changes.rst]: Fixed

**Comment by Stranger6667** [comment]
Probably, we could just ignore the failure or mark it with `allow_failures` on Travis

**Comment by benjaoming** [comment]
Yeah, let's just ignore it and move on. This should be the final 0.13.x release, and since this branch would retain (useless) Python 3.3 support, it doesn't matter that Tox fails to install latest wheel requirement.

**Pull request merged**